packages/documentation/src/pages/forms/creating-a-form-config-file.md

```diff
@@ -22,8 +21,138 @@ The form config itself is an object with many properties that determine how your
+The VA Forms Library code uses the form config to determine which React components to render in order to build your form, usually automatically. React components require props to render properly. These are passed down through the top-level `Form` component, which takes the entire form config itself as a prop. The `Form` component is also connected to the store, so it has access to the form data as well.
 
 For more information, see "[About the component hierarchy](/forms/component-hierarchy)."
+
+## How to structure your form
+
+When creating forms using the VA Forms Library it can be a little confusing figuring out how to structure your code so that it will be easy to maintain and easy to understand later. 
+
+All forms begin with a `form.js` file inside of a `/config` folder. This file is generated by the form generator and is intended to house your `schema` and your `uiSchema`. This structure works well for simple forms, but for more advanced forms we recommend that you keep this file but also break your `schema` and your `uiSchema` across smaller files that are easier to maintain. Here is a rough example of what your `form.js` will look like when you have your `schema` and `uiSchema` inside it:
+
+```js
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/api',
+  trackingPrefix: 'new-form-',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: 'XX-230',
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: 'Please start over to apply for new form benefits.',
+    noAuth: 'Please sign in again to continue your application for new form benefits.'
+  },
+  title: 'My new form',
+  defaultDefinitions: {
+  },
+  chapters: {
+    chapter1: {
+      title: 'Chapter 1',
+      pages: {
+        page1: {
+          path: 'first-page',
+          title: 'First Page',
+          uiSchema: {
+          },
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        }
+      }
+    }
+  }
+};
+```
+
+The area you will focus most of your time on while coding your form will be the `chapters` and `pages` inside the `formConfig` object, this object stores all of the information that the VA Forms Lystem uses to "build" your form. 
+
+The most logical way to break up your code is to create a separate folder for each chapter and then inside that folder to create a folder for each page in that respective chapter with a JavaScript file named for the page, as well as an `index.js` used to import the pages.
+
+```js
+| config
+| -- chapters
+  | --- chapter1
+    |---- index.js
+    |---- page1
+      | ----- page1.js
```

Comment:
Is this the recommended best practices? Seems like a lot of unnecessary nesting to me.